mapping/rights.rb

```diff
@@ -14,27 +14,36 @@ def self.process_metadata(value, *_args)
       date_issued_node = value['date'].find { |n| n.qualifier.field_name.casecmp('date_issued').zero? }
       date_issued = date_issued_node.qualifier.run_method
       rights_uri_node = value['rights'].find { |n| n.qualifier.field_name.casecmp('license').zero? }
-      return nil if rights_uri_node.nil?
+      # rights statement is required and the code below assigns one even it is empty in DSpace
+      if rights_uri_node.nil? && DateTime.parse(date_issued) > DateTime.new(1923, 12, 31)
+        return 'http://rightsstatements.org/vocab/CNE/1.0/'
+      elsif rights_uri_node.nil? && DateTime.parse(date_issued) < DateTime.new(1923, 12, 31)
+        return 'http://rightsstatements.org/vocab/NoC-US/1.0/'
+      end
       rights_uri = rights_uri_node.qualifier.run_method
 
+      # journal article most likely has issued date at year month
+      date_issued << '-31' if date_issued =~ /^\d{4}\-\d{2}$/
```

Comment:
Also these evaluations should happen before the the block above where you are doing `DateTime.parse`